codalab/lib/worksheet_util.py

```diff
@@ -1055,7 +1059,8 @@ def flush_worksheets():
                 else:
                     raise UsageError("unknown directive `%s`" % command)
 
-                raw_to_block.append(None)
+                if not appended_raw_to_block:
```

Comment:
why don't we need to append None if there weren't any schema blocks? (I also can't recall why we appended None before), if you have context can you add a comment here as well why the append None is needed?

Reply:
Based on my understanding, it is because we need a placeholder for other cmds.